component/azstorage/block_blob.go

```diff
@@ -675,15 +676,25 @@ func (bb *BlockBlob) getBlobAttr(blobInfo *container.BlobItem) (*internal.ObjAtt
 	}
 
 	parseMetadata(attr, blobInfo.Metadata)
-	if !bb.listDetails.Permissions {
+	if !bb.listDetails.Permissions && !(mode != 0) {
```

Comment:
!(mode != 0) ?? why double negation in condition check ? does that mean mode == 0